app/models/searchgov_url.rb

```diff
@@ -6,6 +6,7 @@ class SearchgovUrl < ActiveRecord::Base
   MAX_DOC_SIZE = 15.megabytes
   SUPPORTED_CONTENT_TYPES = %w(
                                 text/html
+                                text/plain
                                 application/msword
                                 application/pdf
                                 application/vnd.ms-excel
```

Comment:
Use the braces that are the most appropriate for the various kinds of percent literals. You can refer to here https://github.com/rubocop-hq/ruby-style-guide#percent-s